src/interface/storage.thrift

```diff
@@ -561,9 +561,8 @@ struct LookupAndTraverseRequest {
  */
 
 struct ScanCursor {
-    3: bool                                 has_next,
     // next start key of scan, only valid when has_next is true
```

Comment:
how client knows the scan has get to end？

Reply:
When 'next_cursor' is null.